src/core/quantum_gate.jl

```diff
@@ -92,6 +98,32 @@ q[1]:──X45──
 q[2]:───────
 ```
 
+Lastly, to enable the construction of a `ControlledGate{X45}` made with this 
```

Comment:
If we update the display for controlled gates, this will fall away.

Reply:
removed